src/pages/badge/front.astro

```diff
@@ -17,7 +17,7 @@ const role = Astro.url.searchParams.get("role");
 
 <BadgeLayout>
 	<div class="contents">
-		<div class="more-top-gr3adient"></div>
+		<div class="more-top-gradient"></div>
```

Comment:
Lol, is this unused code?

Reply:
lol no idea, it was there for a bit, it seems